nexus/lib/hdfreader.py

```diff
@@ -367,7 +366,7 @@ def add_group(self,hcur,cur,k,v):
         hcur  = self.hcur[self.ilevel]
         for kr,v in hcur.items():
             k=cur._escape_name(kr)
-            if valid_variable_name(k):
+            if not any([i in ('!"#$%&\'()*+,-./:;<=>?@[\\]^`{|}-\n\t ') for i in k]):
```

Comment:
Revert change.  `valid_variable_name` just added to `developer.py` should be called here.